cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -225,7 +225,12 @@ function move_sriov_pfs_netns_to_node {
   mkdir -p /var/run/netns/
   ln -sf /proc/$pid/ns/net "/var/run/netns/$node"
 
-  local -r sriov_pfs=( /sys/class/net/*/device/sriov_numvfs )
+  local -r sriov_pfs=( $(find /sys/class/net/*/device/sriov_numvfs) )
+  if [ "${#sriov_pfs[@]}" -eq 0 ]; then
+    echo "FATAL: Could not find available sriov PFs"
```

Comment:
You should echo to stderr and not stdout. And in general, in bash, the convention is something like this:
`[ "${#sriov_pfs[@]}" -eq 0 ] && echo "..." .... ; exit 1`

Reply:
About the 2nd part 
I think linter says otherwise (https://github.com/kubevirt/kubevirtci/pull/511), but np i will change now and once linter is in we discuss it again in case needed.